<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IconSelector</class>
 <widget class="QWidget" name="IconSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图标选择</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" colspan="2">
    <widget class="QPushButton" name="pb_custom">
     <property name="text">
      <string>自定义图标</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="le_custom">
     <property name="placeholderText">
      <string>这是自定义图标的路径</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTableWidget" name="tw_default">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>64</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>64</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>64</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>64</number>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pb_ok">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="pb_cancel">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
